src/main/java/com/google/sps/servlets/Statistics.java

```diff
@@ -12,18 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package com.google.sps.servlets.utils;
+package com.google.sps.servlets;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class Statistics {
   // Aggregated score for a list of comments  
-  // Contains sentiment scores in the range [-1, 1] with 0.2 intervals.
-  private List<Double> aggregateValues;
+  // Contains number of sentiment scores in the range [-1, 1] with 0.2 intervals.
+  private List<Integer> aggregateValues;
```

Comment:
I'm not sure if List is the best representation for this bucket counter. Since the boundaries aren't baked into the List object itself, consumers of this Statistics class may not know what element is contained at position i unless they are aware of how the bucketing is defined. Because of this, this design necessitates the use of the bucketing logic in multiple places: 1st to create this Statistics object, and 2nd to know what is contained within this object.

I think a better representation may be a Map.

Have we considered something like Map<Range, Integer>?

Range could be a simple value class like:
```
public final class Range {
  private final inclusiveStart;
  private final exclusiveEnd;
  
  ...
 // Getters + Constructor.
}

```
If we use a Map in this way, then the boundaries would be baked into the Statistics object itself. This would also have the added benefit of making it easier to change how we defined the bucketing policy. I.e. we could begin to use .1 intervals or could begin to use inclusive/exclusive starts/ends.